cilium/cmd/policy_trace.go

```diff
@@ -43,6 +43,7 @@ SOURCE:KEY[=VALUE].
 dports can be can be for example: 80/tcp, 53 or 23/udp.
 If multiple sources and / or destinations are provided, each source is tested whether there is a policy allowing traffic between it and each destination.
 --src-k8s-pod and --dst-k8s-pod requires cilium-agent to be running with disable-endpoint-crd option set to "false".`,
+	Deprecated: "the output is not correct for all policy types. Consider instead using https://app.networkpolicy.io\n",
```

Comment:
This should show up in `Documentation/cmdref/cilium_policy_trace.md`, I think, but it doesn't seem to be the case. Can you please double-check?

Reply:
:facepalm: I was using `make -C Documentation/ cmdref`, not `make -C Documentation/ update-cmdref`.